pkg/types/types.go

```diff
@@ -77,6 +77,7 @@ type VulnerabilityDetail struct {
 	Description      string     `json:",omitempty"`
 	PublishedDate    *time.Time `json:",omitempty"` // Take from NVD
 	LastModifiedDate *time.Time `json:",omitempty"` // Take from NVD
+	Status           bool       `json:",omitempty"` // Rejected or not Rejected cve. Take from NVD
```

Comment:
NVD actually has more statuses like DISPUTED. We currently support "REJECTED" or not, but I want to make it flexible for future enhancement.
https://nvd.nist.gov/general/cve-process

```suggestion
	Status           types.Status       `json:",omitempty"` // Take from NVD
```

Reply:
I mean we should define `type Status string` somewhere.